psycopg/psycopgmodule.c

```diff
@@ -112,6 +112,44 @@ psyco_connect(PyObject *self, PyObject *args, PyObject *keywds)
     return conn;
 }
 
+#define psyco_parse_dsn_doc \
+"parse_dsn(dsn) -- Parse database connection string.\n\n"
```

Comment:
better: parse_dsn(dsn) -> dict. Also I don't think you need the \n's. Even better put all the documentation here and use autodoc to get it in the docs.